repository/RediStick-Core.package/RsRedisEndpoint.class/instance/getJSONAt..st

```diff
@@ -0,0 +1,6 @@
+commands-sets
+getJSONAt: aKey
```

Comment:
'JSON.*' API series should be in other package like 'Redistick-RedisJSON'.  They should be treated as extensions.

Reply:
How about developing 'Redistick-RedisJSON' in your repository first? You can write BaselineOfRedisJSON to load RediStick as a dependency. It would simplify merge problems.

Reply:
yes you're right we need to avoid god class , I will revert this commit